src/web/app/pages-session/session-submission-page/saving-complete-modal/saving-complete-modal.component.html

```diff
@@ -40,6 +43,10 @@ <h5 class="modal-title">
   <p *ngIf="!hasFailToSaveQuestions">
     <b>You are encouraged to download the proof of submission, which will also contain your latest responses.</b>
   </p>
+  <span class="color-red" *ngIf="!hasFailToSaveQuestions && questions.length === 1">
```

Comment:
For this could we use the bootstrap class text-danger to standardise the styling of text.